Metrics card changes (#48)
mulesoft/docs-visualizer: modules/ROOT/pages/visualizer-app-network.adoc

To maximize visualization readability, consumer nodes (nodes that have only outbound connections) are shown at the top. Backend systems (nodes that have only inbound connections) are shown at the bottom.

> When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, it is grouped under the *External Traffic* node.

Reviewer: ```suggestion
When the `ClientId` category is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, they are grouped under the *External Traffic* node.
```

Reviewer: Syntax values should not be used as common nouns, so we need to follow them with a common noun (typically "value" or "option") OR just use the common noun (in this case, "the client ID").
GLOBAL

> The target `application/API/proxy` should have the `ClientId` enforcement policy to detect the `ClientId`.

Reviewer: ```suggestion
The target specified by `application/API/proxy` must have the `ClientId` enforcement policy to detect the `ClientId` value.
```
DITTO

> For usability purposes, client app names replace `ClientIds` in the canvas.

Reviewer: ```suggestion
For usability purposes, client app names replace `ClientIds` values in the canvas.
```

After selecting the organization and environment group, Anypoint Visualizer displays a graph of your application network.

> == View Information About All Services In Your Network

Reviewer: ```suggestion
== View Information About All Services in Your Network
```

The *Monitoring* tab contains:

> * *Metrics summary*: Provides monitoring metrics for the selected node, including Avg response time, Avg throughput, Failures, Avg CPU utilization, and Avg memory utilization. This shows either inbound or outbound values for Avg response time, Avg throughput, and Failures depending on what you’ve selected in the *Metric* card.

Reviewer: GLOBALLY limit list items or steps to one sentence or fragment. If additional info is necessary, move it to a plain text para below that list item:
```suggestion
* *Metrics summary*
Provides monitoring metrics for the selected node, including average response time, average throughput, failures, average CPU utilization, and average memory utilization. This shows either inbound or outbound values for Avg response time, Avg throughput, and Failures, depending on what you’ve selected in the *Metric* card.
```
My approach here is the spell out the type of metric but use UI labels when referring to what is shown in the UI. Might be too granular.
